servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -410,10 +410,6 @@ void RendererSceneRenderRD::environment_set_ssr(RID p_env, bool p_enable, int p_
 	RendererSceneEnvironmentRD *env = environment_owner.getornull(p_env);
 	ERR_FAIL_COND(!env);
 
```

Comment:
So SSR and SSAO were switched on low end here as well. I'm not sure if we will be supporting these in the mobile renderer or not. Do we need `is_ssr_supported` and `is_ssao_supported` methods in `RendererSceneRenderRD` ?

Reply:
We should be able to support both SSR and SSAO in the mobile renderer. They will take a larger proportion of the frame budget than they do on Desktop. But it should be totally feasible.

Reply:
Then we should just leave it as is I guess :)